src/main/java/uk/gov/pay/directdebit/gatewayaccounts/api/GatewayAccountResponse.java

```diff
@@ -52,8 +63,8 @@ public static GatewayAccountResponse from(GatewayAccount gatewayAccount) {
                 gatewayAccount.getPaymentProvider().toString(),
                 gatewayAccount.getDescription(),
                 gatewayAccount.getType().toString(),
-                gatewayAccount.getAnalyticsId()
-        );
+                gatewayAccount.getAnalyticsId(),
+                gatewayAccount.getAccessToken().map(token -> TRUE).orElse(FALSE));
```

Comment:
We can just do `gatewayAccount.getAccessToken().isPresent()`.

Reply:
lol. of course. doh.